mono/eglib/gutf8.c

```diff
@@ -303,7 +303,7 @@ g_utf8_prev_char (const gchar *str)
 	const gchar *p = str;
 	do {
 		p--;
-	} while ((*p & 0xc0) == 0xb0);
+	} while ((*p & 0xc0) == 0x80);
```

Comment:
Please consider deleting the function. The rationale should say that it is incorrect and unused.

Reply:
Remember to get all 3 parts: declaration, definition, and remap.

```
$ git grep _utf8_find_prev_cha
mono/eglib/eglib-remap.h:#define g_utf8_find_prev_char monoeg_utf8_find_prev_char
mono/eglib/glib.h:gchar    *g_utf8_find_prev_char (const char *str, const char *p);
mono/eglib/gutf8.c:g_utf8_find_prev_char (const gchar *str, const gchar *p)

$ git grep g_utf8_prev_char
mono/eglib/eglib-remap.h:#define g_utf8_prev_char monoeg_utf8_prev_char
mono/eglib/glib.h:gchar    *g_utf8_prev_char     (const char *str);
mono/eglib/gutf8.c:g_utf8_prev_char (const gchar *str)
```